Packs/CyrenInboxSecurity/Integrations/CyrenInboxSecurity/CyrenInboxSecurity.py

```diff
@@ -387,6 +394,22 @@ def fetch_incidents(client, client_id, client_secret, last_run,
     return incidents
 
 
+def reset_sample_fetch_command(client):
+    """
+    Reset sample mode to allow next call to fetch-incidents to create a sample incident
+    """
+
+    set_integration_context({"sample_fetched": False})
```

Comment:
You can use the LastRun object in the same way, and then you don't need the `reset_sample_fetch_command`, because you will always can reset the lastRun object by clicking the `reset lastRun` link in the instance configuration (attached is a screenshot from a random integration instance with fetch).
![image](https://user-images.githubusercontent.com/59408745/133208988-c68c3016-11c1-4b4c-9cc1-52c85b739ae3.png)

Reply:
done